### ISSUE TITLE
no-multiple-empty-lines does not report any errors if file contains interpolated strings

### ISSUE THREAD

**Issue by Wilfred** [eslint/eslint#7312]
**Tell us about your environment**

* **ESLint Version:** 3.7.1
* **Node Version:** 6.4.0
* **npm Version:** 3.10.3

**What parser (default, Babel-ESLint, etc.) are you using?**

Babel, see full configuration

**Please show your full configuration:**

```json
{
    "parserOptions": {
        "ecmaVersion": 6,
        "ecmaFeatures": {
            "jsx": true,
            "impliedStrict": true
        },
        "sourceType": "module"
    },
    "rules": {
        "no-multiple-empty-lines": ["warn", {"max": 1}]
    },
    "plugins": [
        "react"
    ]
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
import './baz';


function foo() {
  `${s}-foo`;
}

function bar() {
  `${x} bar`;
}
```

I ran eslint as follows:

```bash
$ eslint static/dodgy_direct.jsx --ext .jsx -c .eslint.format.json --no-eslintrc --fix
```

**What did you expect to happen?**

Duplicate blank line should be removed, so the correct output should be:

```js
import './baz';

function foo() {
  `${s}-foo`;
}

function bar() {
  `${x} bar`;
}
```

**What actually happened? Please include the actual, raw output from ESLint.**

File was unchanged, with no errors reported.

**Comment by not-an-aardvark** [comment]
Thanks for the report. I can reproduce this issue.

(To clarify, I don't think the problem is with the autofixer in particular -- the rule simply isn't reporting an error at all with that code, even when the `--fix` flag is omitted.)

**Comment by platinumazure** [comment]
For my sanity, could we definitively clarify whether the issue is "rule doesn't report" vs "rule reports but doesn't fix", and edit that into the title and original post? Thanks!
